config/default.php

```diff
@@ -17,7 +17,7 @@
  */
 $config['debug'] = false;
 
-/**
+/*
  * Enable the YouTube signature decipher function
  * WARNING: This downloads javascript code from a 3rd party server and interprets it!
  * This MAY harm your server, if the 3rd party server delivers malicious code!
```

Comment:
I think it would be impossible to execute malicious code with the current decipher script. The current decipher didn't execute the raw code instead using matched command dictionary to reverse, splice, or swapping signature arrays..